docs/reference/transactions.md

```diff
@@ -114,6 +114,17 @@ In the case that this example runs successfully, then both database writes are c
         )
     ```
 
+## Isolation levels
+
+By default, Prisma sets the isolation level to the value currently configured in the database. You can modify this
+default with the `isolation_level` argument (see [supported isolation levels](https://www.prisma.io/docs/orm/prisma-client/queries/transactions#supported-isolation-levels)).
+
+```py
+prisma.tx(
+    isolation_level=prisma.TransactionIsolationLevel.READ_UNCOMMITTED,
+)
```

Comment:
nit: I think it would be helpful to include imports for this example & I think it's slightly incorrect as `prisma` can't be both the client variable and the module
```suggestion
from prisma import TransactionIsolationLevel

client.tx(
    isolation_level=TransactionIsolationLevel.READ_UNCOMMITTED,
)
```
The variable shadowing issue is also why I tend to try and stick to either using `db` or `client` as the `Prisma` instance, or to include a more complete example

```suggestion
from prisma import Prisma, TransactionIsolationLevel

client = Prisma()
client.tx(
    isolation_level=TransactionIsolationLevel.READ_UNCOMMITTED,
)
```
Up to you which you think is better :)